Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/events/TraceLocation.kt

```diff
@@ -1,9 +1,11 @@
 package de.rki.coronawarnapp.eventregistration.events
 
+import android.os.Parcelable
 import de.rki.coronawarnapp.server.protocols.internal.pt.TraceLocationOuterClass
+import okio.ByteString
 import org.joda.time.Instant
 
-interface TraceLocation {
+interface TraceLocation : Parcelable {
```

Comment:
I don't think `Parcelable` should be here. More important that we keep things related to the context where it is used. Parcelable can be used in a service ,but we don't have a service and we will not. So I would like to keep things where they only belong

Reply:
> If you want to have DefaultTraceLocation as Parcelable, Please remove the interface I think it is not needed and don't really know why we create interface to be implemented by data class.

I don't think it makes sense two have two data classes just to have one of them parcelable, if both are the same like they currently are. If this becomes an issue we can split them. The interface/data class separation was created by @LukasLechnerDev, Interfaces were likely created during initial setup so everyone can start working on it. I will ask him.

Reply:
Yes, I think we can remove the interface. We created it initially and I created the data class for it to stay compatible. 

Maybe we can create already data classes instead of interfaces next time at feature start?

Reply:
I've refactored this and removed the interface.